weed/server/filer_server.go

```diff
@@ -115,6 +116,10 @@ func NewFilerServer(defaultMux, readonlyMux *http.ServeMux, option *FilerOption)
 	domains := strings.Split(allowedOrigins, ",")
 	option.AllowedOrigins = domains
 
+	v.SetDefault("filer.expose_directory_metadata.enabled", true)
```

Comment:
@chrislusf Is this the option you are referring to in your comment above?

Reply:
this file https://github.com/seaweedfs/seaweedfs/blob/e88467bb36b3618723c663d6b0e96541f654371c/weed/command/server.go#L109

Reply:
Thank you, updated it.